test/NuGet.Core.Tests/NuGet.Build.Tasks.Test/GetRestoreProjectStyleTaskTests.cs

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using NuGet.Configuration;
+using NuGet.ProjectModel;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Build.Tasks.Test
+{
+    public class GetRestoreProjectStyleTaskTests
+    {
+        [Fact]
+        public void GetRestoreProjectStyleTask_WhenInvalidProjectStyleSupplied_LogsError()
```

Comment:
It should be `<MemberName>` not `<TypeName>`.  All these tests are in a class which clarifies which type (`GetRestoreProjectStyleTask`) is being tested.  So:  `Execute_WhenInvalidProjectStyleSupplied_LogsError`